.github/workflows/_e2e_tests.yml

```diff
@@ -37,6 +37,7 @@ jobs:
           ADMIN_UVICORN_NUM_WORKERS: "2"
           ADMIN_UVICORN_PORT: "8081"
           COMMON_LOG_LEVEL: "DEBUG"
+          COMMON_CONTENT_MAX_SIZE : "10000000"
```

Comment:
Could you use "10_000_000" instead? It's easier to read and avoids errors

https://peps.python.org/pep-0515/

Reply:
Done